conf/checksums

```diff
@@ -1,5 +1,7 @@
+accumulo:2.1.0:9bbc4defc114013f145e9e9fdd08683c842a29faaa01ea6e80555549a6aecef86ee6657cce23183b411a64c31123db682a6944e2825eef83c76fb5a91620235f
 accumulo:2.0.1:b443839443a9f5098b55bc5c54be10c11fedbaea554ee6cd42eaa9311068c70bd611d7fc67698c91ec73da0e85b9907aa72b98d5eb4d49ea3a5d51b0c6c5785f
 accumulo:1.10.2:9b6e9286133120588f4c682e3fadee33704cf5983ef7e3c0be84c1678bd9bcf283a2c6efa6d4074c73f30032f5c0aa1073c15ec39f90d215505e028a1cf0a739
+hadoop:3.3.4:ca5e12625679ca95b8fd7bb7babc2a8dcb2605979b901df9ad137178718821097b67555115fafc6dbf6bb32b61864ccb6786dbc555e589694a22bf69147780b4
 hadoop:3.3.3:1f5762682cef3daff8b2379fe7e40efca107bb7e8dcaa4a513e3bc0c082067759dd05d493ec997433dde2c89ea63dbc93aee0bba60045f89d3ec2d3f687f58b3
```

Comment:
The 3.3.3 line can be deleted. We don't need to retain earlier versions in the same release line.

```suggestion
hadoop:3.3.4:ca5e12625679ca95b8fd7bb7babc2a8dcb2605979b901df9ad137178718821097b67555115fafc6dbf6bb32b61864ccb6786dbc555e589694a22bf69147780b4
```